core/deployment/src/main/java/io/quarkus/deployment/ide/IdeConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.deployment.ide;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot
+public class IdeConfig {
```

Comment:
not sure whre the bug is but using dev config editor it makesthis "quarkus.ide.target=code' which gives an error.


also i believe the current property is "quarkus.ide" which i think is limiting our options.

i would suggest "quarkus.ide.command" or similar allowing us to do i.e

```
quarkus.ide.command=custom
quarukus.ide.custom.command=open -a /Applications/Eclipse.app {filename}"

or similar in the future.